src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -328,6 +330,8 @@ subroutine InitAllocate(this, bounds)
       ! statements.
       allocate(this%vcmx25_z_patch  (begp:endp,1:nlevcan)) ; this%vcmx25_z_patch    (:,:) = 30._r8
       allocate(this%jmx25_z_patch   (begp:endp,1:nlevcan)) ; this%jmx25_z_patch     (:,:) = 60._r8 
+      allocate(this%vcmx_prevyr     (begp:endp,1:nlevcan)) ; this%vcmx_prevyr       (:,:) = 85._r8
+      allocate(this%jmx_prevyr      (begp:endp,1:nlevcan)) ; this%jmx_prevyr        (:,:) = 50._r8
```

Comment:
hopefully  doesn't  really matter, but the hard coded values we're replacing  in  LUNA are  were for `vcmx25` = 50 (not 85) & `jmx25` = 85 (not 50)
In lines 331-2 of photosynthesisMod `vcmx25` and  `jmx25` are initialized to 30 & 60, respectively.  Either one of these seem preferable, but not sure which is better?

Reply:
Thank you for raising this point here @wwieder . In addition to cold start, times I can think of when these initial values will matter are (1) in the first year of existence of a crop that newly appears in a grid cell in a transient case; and (2) in the first year after restart for cases using old restart files, including all cases run from master until we update our out-of-the-box finidat files. So, while these values may not be hugely important, they seem more important than your typical cold start initialization value.

Reply:
See also comment from @lmbirch89: https://github.com/ESCOMP/CTSM/pull/947#discussion_r407784297

Reply:
Since, our simulations have been run with these values -- we'll need to get the correct values on #947.

Reply:
We'll fix this for #1107

Reply:
This is coming in with my PR #1102